tests/unit/test_validate_attribute.py

```diff
@@ -99,6 +99,53 @@
     }
 )
 
+TEST_DF_FILEVIEW = DataFrame(
```

Comment:
Thanks for all the test cases here. I am curious: what if the entity id is missing and also the manifest file path does not exist, what would happen? I think based on the current design, only one message gets raised?

Reply:
That is correct, only one message can be raised at present. In this case, the `missing entityId` error would be raised. [We give precedence](https://sagebionetworks.slack.com/archives/C07HLTBS5D2/p1726001699040209?thread_ts=1726000362.049489&cid=C07HLTBS5D2) to the `entityId` for these cases.

Reply:
Thanks for the clarification Brad. Can we ask you to document all these expected behavior somewhere with examples? I will help coordinate and see where would be a good place to document.

Reply:
Sure! Just let me know where would be best.

Reply:
Let me know if you need a new ticket for the documentation work.

Reply:
It's OK I'll do it as part of this ticket

Reply:
I have updated the documentation